Formula/node.rb

```diff
@@ -93,21 +85,21 @@ def post_install
     # Kill npm but preserve all other modules across node updates/upgrades.
     rm_rf node_modules/"npm"
 
-    cp_r libexec/"npm/lib/node_modules/npm", node_modules
+    cp_r libexec/"lib/node_modules/npm", node_modules
     # This symlink doesn't hop into homebrew_prefix/bin automatically so
-    # remove it and make our own. This is a small consequence of our bottle
-    # npm make install workaround. All other installs **do** symlink to
-    # homebrew_prefix/bin correctly. We ln rather than cp this because doing
-    # so mimics npm's normal install.
-    ln_sf npm_exec, "#{HOMEBREW_PREFIX}/bin/npm"
+    # we make our own. This is a small consequence of our bottle npm &
```

Comment:
Which part? There's the start of the sentence cut off above the diff, which might not be helping.

```ruby
    # This symlink doesn't hop into homebrew_prefix/bin automatically so
    # we make our own. This is a small consequence of our bottle npm &
    # retain a private copy in libexec setup. All other installs **do**
    # symlink to homebrew_prefix/bin correctly. We ln rather than cp this
    # because doing so mimics npm's normal install.
```